src/main/java/com/khutircraftubackend/product/ProductController.java

```diff
@@ -27,9 +27,10 @@ public ResponseEntity<ProductEntity> createProduct(
             @ModelAttribute("image") MultipartFile image,
             @ModelAttribute("available") Boolean available,
             @ModelAttribute("description") String description,
+            @ModelAttribute("categoryId") Long categoryId,
```

Comment:
may i suggest to refactor it similar as it's been done in CategoryController in `createCategory()` method:
e.g.:
```java
@PostMapping("/")
@ResponseStatus(HttpStatus.CREATED)
public ProductResponse createProduct(
   @ModelAttribute CreateProductRequest request,
   @RequestPart(value = "thumbnailImage", required = false) MultipartFile thumbnailImage
   ... 
) ... { ... }
```

in this case, do you think we could generalize `ProductResponse` into something standard, 
that could probably be shared between multiple controllers?

Reply:
Refacted it similar in CategoryController. Created generalize ProductResponse.